src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -112,27 +112,27 @@ private void ResumeRestartIfScheduled()
                 using (System.Threading.ExecutionContext.SuppressFlow())
                 {
                     _typedLogger.LogDebug("Resuming scheduled restart.");
-                    Task.Run(async () => await ScheduleRestartAsync());
+                    Task.Run(async () => await ScheduleRestartAsync("Running restart requested while in a restart suspension scope."));
```

Comment:
I find this a little hard to read. How about just using the log from above "Resuming scheduled restart." in both places? for a lot of these, like my other comment, it looks like we already have separate logs describing the reason for the restart.